website/addons/box/settings/defaults.py

```diff
@@ -6,3 +6,5 @@
 
 BOX_OAUTH_TOKEN_ENDPOINT = 'https://www.box.com/api/oauth2/token'
 BOX_OAUTH_AUTH_ENDPOINT = 'https://www.box.com/api/oauth2/authorize'
+
+BOX_SHARE_URL_TEMPLATE = 'https://app.box.com/files/0/f/{0}'
```

Comment:
This does not need to go in the settings file since it is not meant to be configured. This can be moved to `config.py`.